examples/interactive_local_hyperdrive_example.py

```diff
@@ -1,118 +1,228 @@
-"""Example script for using interactive hyperdrive to launch a local chain."""
+"""An example of using agent0 to simulate hyperdrive on a local chain."""
 
 # %%
-# Variables by themselves print out dataframes in a nice format in interactive mode
-# pylint: disable=pointless-statement
-
 import datetime
 
 from fixedpointmath import FixedPoint
 
-from agent0 import LocalChain, LocalHyperdrive
+from agent0 import LocalChain, LocalHyperdrive, PolicyZoo
+
+# %% [markdown]
+#####################
+# Initialization
+#####################
 
 # %%
 # Parameters for local chain initialization, defines defaults in constructor
 local_chain_config = LocalChain.Config()
+
 # Launches a local chain in a subprocess
 # This also launches a local postgres docker container for data under the hood, attached to the chain.
-# Each hyperdrive pool will have it's own database within this container
-# NOTE: LocalChain is a subclass of Chain
 chain = LocalChain(local_chain_config)
-# Can connect to a specific existing chain
-# existing_chain = Chain("http://localhost:8545")
 
+# Initialize agents from the chain
 # %%
-# Initialize the interactive object with specified initial pool parameters and the chain to launch hyperdrive on
-# An "admin" user (as provided by the Chain object) is launched/funded here for deploying hyperdrive
+# Generate funded trading agents from the interactive object
+# Names are reflected on output data frames and plots later
+# NOTE a local chain can initialize with base and eth.
+# Base is a mock token used for all underlying pools deployed.
+agent0 = chain.init_agent(
+    eth=FixedPoint(100),
+    name="agent0",
+)
+# We can initialize an agent with a custom policy - more on that later
+agent1 = chain.init_agent(
+    eth=FixedPoint(100),
+    name="agent1",
+    policy=PolicyZoo.random,
+    policy_config=PolicyZoo.random.Config(),
+)
+
+# %%
+# Initialize the interactive object with specified initial pool parameters and the chain to launch hyperdrive on.
```

Comment:
This first sentence reads weird to me. Maybe:

`# Deploy a specified pool parameters on a specified chain, return an initialized LocalHyperdrive object.`